org.wabarc.wayback.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>org.wabarc.wayback</id>
```

Comment:
`wabarc.org` doesn't exists, so the app-id doesn't meet the requirements.

Reply:
why did you close the PR?

Reply:
the branch name has always been irrelevant, it didn't need changing. this just make reviewer life harder as they have to repeat themselves (and lost the history of the review)